docs/preview_features/build.md

```diff
@@ -0,0 +1,115 @@
+
+## Introduction
+Sometimes you might need to build packages from source. This can be due to a variety of reasons, such as:
+
+1. You are using the package for local development, and want to install and build the package into your workspace.
+2. You want to use source for a package that is not available on conda-forge.
+3. You want to build your package locally into a conda package.
+
+We've been working to support these use-cases with the `build` feature in pixi. 
+The vision is to enable building of packages from source, for any language, on any platform.
+
+??? note "Preview feature"
+    This feature is currently in preview phase. To enable use the [preview feature](../reference/pixi_manifest.md#preview-features).
+    ```toml
+    [project]
+    # .. other configuration
+    preview = ["build"]
+    ```
+
+## Limitations
+Currently, the `build` feature has a number of limitations:
+
+1. Recursive source dependencies are not supported.
+2. Workspace dependencies cannot be inherited.
+3. Progress reporting should be improved.
+4. Only a limited number of languages and build systems are supported.
+5. Missing parametrization of build-systems.
```

Comment:
Bas did implement arguments of some sort already right?

Reply:
Indeed, let me rephrase.